.github/workflows/ci.yml

```diff
@@ -25,15 +25,15 @@ jobs:
         run: npm install
 
       - name: Lint Files (eslint)
-        uses: trunk-io/trunk-action@v1
+        uses: trunk-io/trunk-action@v1.1.17
         with:
 
           # Run on everything except the docs folder.
           arguments: --ignore=docs/** --filter=eslint
           check-mode: all
 
       - name: Lint Files (other)
-        uses: trunk-io/trunk-action@v1
+        uses: trunk-io/trunk-action@v1.1.17
```

Comment:
`trunk-io/trunk-action@v1.1.18` is causing build errors in some pull requests, apparently because there are two steps trying to upload an artifact with the same name in a job. Examples:

* https://github.com/eslint/eslint/actions/runs/11528700815/job/32096167483
*  https://github.com/eslint/eslint/actions/runs/11528810311/job/32100020453
*  https://github.com/eslint/eslint/actions/runs/11529133487/job/32097195940

This was working well in v1.1.17 because the artifacts would have different names. I did some tests on a fork of `eslint` and the build errors in the PRs disappeared after pinning the `trunk-io/trunk-action` dependency in the main branch and rebasing the PRs.

Reply:
Should we maybe do the same in a few other places where we use `trunk-io/trunk-action`?

https://github.com/eslint/eslint/blob/425202ed49a1372c1719d4e7b48d0fbdda8af9fa/.github/workflows/ci.yml#L49

https://github.com/eslint/eslint/blob/425202ed49a1372c1719d4e7b48d0fbdda8af9fa/.github/workflows/ci.yml#L57

https://github.com/eslint/eslint/blob/425202ed49a1372c1719d4e7b48d0fbdda8af9fa/.github/workflows/annotate_pr.yaml#L23

Reply:
Thanks! Done in 31f5c87f97286d74d8324de5fbeb75f7c3998851.